File: TemplaFront/src/app/componentes/models/ReporteStockBajoDTO.ts
export interface ReporteStockBajoDTO {
  nombreProducto: string;
  tipoProducto: TipoProducto;
  unidadMedida: UnidadMedida;
  stockActual: number;
  stockMinimo: number;
  cantidadFaltante: number;
  activo: boolean;
}

export enum TipoProducto {
  INSUMO = 'INSUMO',
  ACOMPAÑANTE = 'ACOMPAÑANTE',
  BEBIDA = 'BEBIDA'
}

export enum UnidadMedida {
  KILOGRAMO = 'KILOGRAMO',
  LITRO = 'LITRO',
  GRAMO = 'GRAMO',
  UNIDAD = 'UNIDAD'
}

File: BackEnd/TemplaRestaurant/src/main/java/Templa/Tesis/App/dtos/ReporteStockBajoDTO.java
package Templa.Tesis.App.dtos;

import Templa.Tesis.App.Enums.TipoProducto;
import Templa.Tesis.App.Enums.UnidadMedida;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ReporteStockBajoDTO {
    private String nombreProducto;
    private TipoProducto tipoProducto;
    private UnidadMedida unidadMedida;
    private Double stockActual;
    private Double stockMinimo;
    private Double cantidadFaltante;
    private Boolean activo;
}
